src/core/src/type/element_type.cpp

```diff
@@ -282,6 +287,10 @@ Type fundamental_type_for(const Type& type) {
         return from<element_type_traits<Type_t::f32>::value_type>();
     case Type_t::f64:
         return from<element_type_traits<Type_t::f64>::value_type>();
+    case Type_t::f8e4m3:
+        return from<element_type_traits<Type_t::f8e4m3>::value_type>();
+    case Type_t::f8e5m2:
+        return from<element_type_traits<Type_t::f8e5m2>::value_type>();
```

Comment:
don't see `nf4` here

Reply:
Yes, and looks like in this file there are more places where `nf4` should be added. 

As this lack of `nf4` is not result of this PR maybe update missing part in separate and here focus on `float8`.

Reply:
I've updated the `nf4` as it looks simple change.